drivers/adodb-mssqlnative.inc.php

```diff
@@ -1137,9 +1137,9 @@ function _fetch($ignore_fields=false)
 	{
 		if ($this->fetchMode & ADODB_FETCH_ASSOC) {
 			if ($this->fetchMode & ADODB_FETCH_NUM)
-				$this->fields = @sqlsrv_fetch_array($this->_queryID,SQLSRV_FETCH_BOTH);
+				$this->fields = sqlsrv_fetch_array($this->_queryID,SQLSRV_FETCH_BOTH);
 			else
-				$this->fields = @sqlsrv_fetch_array($this->_queryID,SQLSRV_FETCH_ASSOC);
+				$this->fields = sqlsrv_fetch_array($this->_queryID,SQLSRV_FETCH_ASSOC);
```

Comment:
$this->fields = sqlsrv_fetch_array($this->_queryID, ($this->fetchMode & ADODB_FETCH_NUM ? SQLSRV_FETCH_BOTH : SQLSRV_FETCH_ASSOC));

Reply:
Well, you're right, but my focus is on the `@`-Operator and any ideas/problems with this.
I may remove the `if`/`else` condition later.